assets/src/dashboard/app/views/exploreTemplates/content/index.js

```diff
@@ -79,6 +79,14 @@ function Content({
                       'No more templates',
                       'web-stories'
                     )}
+                    allDataLoadedAriaMessage={__(
+                      'All templates are loaded',
+                      'web-stories'
+                    )}
+                    loadingAriaMessage={__(
+                      'Loading more templates',
+                      'web-stories'
+                    )}
```

Comment:
since templates are internal right now so there's never actually more to load, I mostly added this for continuity (same with saved templates)